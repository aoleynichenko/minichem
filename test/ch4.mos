[Molden Format]
[Atoms] Angs
  C   1   6    0.00000000    0.00000000    0.00000000
  H   2   1    1.18693526    1.18693526   -1.18693526
  H   3   1   -1.18693526   -1.18693526   -1.18693526
  H   4   1    1.18693526   -1.18693526    1.18693526
  H   5   1   -1.18693526    1.18693526    1.18693526
[5D]
[GTO]
   1
 s  3
 7.16168370e+01  1.54328970e-01
 1.30450960e+01  5.35328140e-01
 3.53051220e+00  4.44634540e-01
 s  3
 2.94124940e+00  -9.99672300e-02
 6.83483100e-01  3.99512830e-01
 2.22289900e-01  7.00115470e-01
 p  3
 2.94124940e+00  1.55916270e-01
 6.83483100e-01  6.07683720e-01
 2.22289900e-01  3.91957390e-01

   2
 s  3
 3.42525091e+00  1.54328970e-01
 6.23913730e-01  5.35328140e-01
 1.68855400e-01  4.44634540e-01

   3
 s  3
 3.42525091e+00  1.54328970e-01
 6.23913730e-01  5.35328140e-01
 1.68855400e-01  4.44634540e-01

   4
 s  3
 3.42525091e+00  1.54328970e-01
 6.23913730e-01  5.35328140e-01
 1.68855400e-01  4.44634540e-01

   5
 s  3
 3.42525091e+00  1.54328970e-01
 6.23913730e-01  5.35328140e-01
 1.68855400e-01  4.44634540e-01

[MO]
Sym=C1
Ene=  -300.1446932
Spin=Alpha
Occup=2.00000000
   1  -0.99194843
   2  -0.03812670
   3  -0.00000000
   4  -0.00000000
   5  -0.00000000
   6   0.00694779
   7   0.00694779
   8   0.00694779
   9   0.00694779
Sym=C1
Ene=   -24.7505642
Spin=Alpha
Occup=2.00000000
   1  -0.22124434
   2   0.62894724
   3  -0.00000000
   4   0.00000000
   5  -0.00000000
   6   0.18070622
   7   0.18070622
   8   0.18070622
   9   0.18070622
Sym=C1
Ene=   -14.1108486
Spin=Alpha
Occup=2.00000000
   1  -0.00000000
   2  -0.00000000
   3  -0.09103036
   4   0.54927728
   5   0.12833711
   6   0.17398576
   7  -0.30934899
   8  -0.26999979
   9   0.40536303
Sym=C1
Ene=   -14.1108486
Spin=Alpha
Occup=2.00000000
   1  -0.00000000
   2   0.00000000
   3  -0.54058947
   4  -0.12207032
   5   0.13901202
   6  -0.42278062
   7   0.27615806
   8  -0.14740471
   9   0.29402727
Sym=C1
Ene=   -14.1108486
Spin=Alpha
Occup=2.00000000
   1   0.00000000
   2  -0.00000000
   3  -0.16105582
   4   0.09927627
   5  -0.53913609
   6   0.25174528
   7   0.31690710
   8  -0.42161849
   9  -0.14703389
Sym=C1
Ene=    19.4528153
Spin=Alpha
Occup=0.00000000
   1  -0.00000000
   2   0.00000000
   3  -0.09575851
   4   1.07487155
   5  -0.22806529
   6  -0.66938723
   7   0.41646023
   8   0.77558450
   9  -0.52265750
Sym=C1
Ene=    19.4528153
Spin=Alpha
Occup=0.00000000
   1   0.00000000
   2  -0.00000000
   3  -1.06299564
   4  -0.03265628
   5   0.29241436
   6   0.76969062
   7  -0.44539978
   8   0.40918359
   9  -0.73347443
Sym=C1
Ene=    19.4528153
Spin=Alpha
Occup=0.00000000
   1   0.00000000
   2  -0.00000000
   3  -0.27821376
   4  -0.24518776
   5  -1.03875521
   6  -0.28576654
   7  -0.86622476
   8   0.59430876
   9   0.55768254
Sym=C1
Ene=    20.5079753
Spin=Alpha
Occup=0.00000000
   1  -0.25187319
   2   1.62003754
   3  -0.00000000
   4  -0.00000000
   5  -0.00000000
   6  -0.66442178
   7  -0.66442178
   8  -0.66442178
   9  -0.66442178

