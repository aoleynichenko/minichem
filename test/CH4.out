Minichem 0.0
By Alexander Oleynichenko, 2016
Build:      1281
Build date: Mar  9 2016 21:44:01
Compiler:   gcc 5.2.0
Date:       Wed Mar  9 22:48:16 2016
Cores:      1
Arguments:  ../bin/minichem CH4.nw 
------------------------------------------------------------
>>> CH4.nw
# Hartree-Fock calculation of CH4

start CH4
memory 400 mb
echo

geometry
C   0.000000000000000   0.000000000000000   0.000000000000000
H   0.628099138030795   0.628099138030795  -0.628099138030795
H  -0.628099138030795  -0.628099138030795  -0.628099138030795
H   0.628099138030795  -0.628099138030795   0.628099138030795
H  -0.628099138030795   0.628099138030795   0.628099138030795
end

basis "ao basis" SPHERICAL
H    S
      3.42525091             0.15432897       
      0.62391373             0.53532814       
      0.16885540             0.44463454  
C    S
     71.6168370              0.15432897       
     13.0450960              0.53532814       
      3.5305122              0.44463454       
C    S
      2.9412494             -0.09996723
      0.6834831              0.39951283    
      0.2222899              0.70011547            
C    P
      2.9412494              0.15591627       
      0.6834831              0.60768372       
      0.2222899              0.39195739    
end

out
  molden
end

task scf
------------------------------------------------------------
                    Basis information
   H
  ----
  1   1S     3.42525091     0.15432897
             0.62391373     0.53532814
             0.16885540     0.44463454


   C
  ----
  1   1S    71.61683700     0.15432897
            13.04509600     0.53532814
             3.53051220     0.44463454

  2   2S     2.94124940    -0.09996723
             0.68348310     0.39951283
             0.22228990     0.70011547

  3   2P     2.94124940     0.15591627
             0.68348310     0.60768372
             0.22228990     0.39195739


Spherical basis set summary:
*---------*-----------------*--------*------------*
| Element |   Description   | Shells | Primitives |
*---------*-----------------*--------*------------*
|   H     | ao basis        |   1    |     3      |
|   C     | ao basis        |   3    |     9      |
*---------*-----------------*--------*------------*

------------------------------------------------------------
          ********************************
          *      Parallel SCF Module     *
          ********************************

Formula:   
Atoms:     5
Electrons: 10
Mol mass:  16.043
Charge:    0
Spin mult: 1
Units:     angstroms
------------------------------------------------------------
One-electron integrals evaluation algorithm: Obara-Saika
Two-electron integrals evaluation algorithm: Obara-Saika
One-electron integrals done in 0.000001 sec

****************** BASIS ORTHOGONALIZATION *******************
Algorithm: canonical
Basis functions elimination threshold: 1e-08
Overlap matrix lowest eigenvalue = 0.21959790
AO basis orthogonalization done in 0.000083 sec

Initial guess: extended Huckel theory
Initial energy = -52.86828401
Initial guess done in 0.003529 sec

#bfns = 9
#eris = 1035

 iter.       Energy         Delta E       RMS-Dens       DIIS-Err     time
----------------------------------------------------------------------------
   1     -40.17939085    -0.77243642     0.37680873     0.33152279    0.03
   2     -39.59909566     0.58029519     0.09494761     0.02005333    0.05
   3     -39.73858027    -0.13948461     0.02151056     0.00181263    0.08
   4     -39.72666288     0.01191739     0.00337264     0.00054865    0.11
   5     -39.72678344    -0.00012056     0.00002831     0.00000153    0.14
   6     -39.72678335     0.00000009     0.00000001     0.00000000    0.17
----------------------------------------------------------------------------
          Total SCF energy =   -39.72678335
  Nuclear repulsion energy =    13.46132959


      Time for:           sec
---------------------------------
  Orthogonalization      0.000
  Initial guess          0.004
  Density matrix         0.022
  Fock matrix            0.129
  Diagonalization        0.000
---------------------------------


         Molecular Orbitals Summary
  +-----+-----+----------------+----------+
  | No  | Occ |     Energy     | Symmetry |
  +-----+-----+----------------+----------+
  | 1   |  2  |   -11.03011002 |     ?    |
  | 2   |  2  |    -0.90956613 |     ?    |
  | 3   |  2  |    -0.51856393 |     ?    |
  | 4   |  2  |    -0.51856393 |     ?    |
  | 5   |  2  |    -0.51856393 |     ?    |
  | 6   |  0  |     0.71487752 |     ?    |
  | 7   |  0  |     0.71487752 |     ?    |
  | 8   |  0  |     0.71487752 |     ?    |
  | 9   |  0  |     0.75365392 |     ?    |
  +-----+-----+----------------+----------+

        Mulliken population analysis
        ----------------------------

      Atom     Population     Charge
      1  C     6.25724858  -0.25724858
      2  H     0.93568785   0.06431215
      3  H     0.93568785   0.06431215
      4  H     0.93568785   0.06431215
      5  H     0.93568785   0.06431215


        Loewdin population analysis
        ---------------------------

      Atom     Population     Charge
      1  C     6.14051085  -0.14051085
      2  H     0.96487229   0.03512771
      3  H     0.96487229   0.03512771
      4  H     0.96487229   0.03512771
      5  H     0.96487229   0.03512771

               Multipole moments
               -----------------
  D       x            y            z
     0.00000000  -0.00000000  -0.00000000

  |D| = 0.00000000 a.u. = 0.00000000 Debye


Allocated: 51820 bytes
Freed:     44972 bytes
Max usage: 15200 bytes
Total time: 0.18 sec
